SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LegacyLoadBalancingStrategy.cpp

```diff
@@ -227,6 +322,21 @@ void FLegacyLoadBalancing::Init(ISpatialOSWorker& Connection, FLoadBalancingShar
 		OutServerWorkerData.Add(ServerWorkerCustomAssignment.Get());
 	}
 #endif //! UE_BUILD_SHIPPING
+
+	OutLoadBalancingData.Add(AlwaysRelevantStorage.Get());
+	OutLoadBalancingData.Add(ServerAlwaysRelevantStorage.Get());
+
+#ifndef BENCH_INTEREST_PERF
+	const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+	if (Settings->bUserSpaceServerInterest)
+	{
+		InterestManager =
+			MakeUnique<FInterestManager>(SharedData->InterestF, *PositionStorage, *AlwaysRelevantStorage, *ServerAlwaysRelevantStorage);
+	}
+#else
+	InterestManager = MakeUnique<FInterestManager>(SharedData->InterestF, *new DbgPositionStorage(TestBox, 210000),
+												   *new FAlwaysRelevantStorage, *new FServerAlwaysRelevantStorage);
```

Comment:
new FServerAlwaysRelevantStorage seems sus.

Reply:
It's only used in the local benchmark tests